.gitignore

```diff
@@ -1,5 +1,3 @@
-pkg/registry/build_registry

```

Comment:
We still need this. Let's say user run build_registry.go to generate registry go files on his local setup, we do not want him to checkin the binary.